ee/localserver/request-id_test.go

```diff
@@ -0,0 +1,101 @@
+package localserver
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.etcd.io/bbolt"
+)
+
+func Test_localServer_requestIdHandler(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name string
+	}{
```

Comment:
Do you see this as getting other input possibilities? Is the table driven tests adding anything?

Reply:
just consistency .... I'm so used to table driven tests now that when I come across one that is not it throws me off and I have to spend extra time parsing it. Perhaps down the road we'll come up with the need to mock the inputs to the test functions or increase complexity for some reason.